src/components/login-form.js

```diff
@@ -1,14 +1,110 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import VoteForm from './vote-form';
 
 export default function LoginForm() {
-  return <Wrapper>안녕 나는 로그인 폼!</Wrapper>;
+	const [logForm, setLogForm] = useState({ email: '', password: '' });
+	const [isloged, setloged] = useState(false);
+
+	const erasePW = () => {
+		document.getElementById('password').value = '';
+	};
+
+	const eraseEmail = () => {
+		document.getElementById('email').value = '';
+	};
+
+	const handleFormChange = (e) => {
+		setLogForm({ ...logForm, [e.target.id]: e.target.value });
+	};
+
+	const handleSubmit = (logForm) => {
+		const { email, password } = logForm;
+		if (email === '' || password === '') {
+			alert('모든 항목을 입력해주세요!');
+			return false;
+		} else {
+			axios
```

Comment:
```suggestion
		if (email === '' || password === '') {
			alert('모든 항목을 입력해주세요!');
			return false;
		} 
		
		axios
```
위의 if문에 해당된 경우 return문이 실행되며 함수가 종료되기 때문에, 이후 실행되는 line들은 모두 해당 조건에 충족되지 않는다고 봐도 되겠죠? 
예시 참고해서 리팩토링 해보세요 ㅎㅎ! [예시링크](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html)
[더 참고할만한 글](https://chchoing88.github.io/ho_blog/refactoring/concise-conditionals/)